scripts/summary.py

```diff
@@ -127,7 +140,7 @@ def get_assembly_pass(percent_coverage):
     columns.sort()
     primary_columns = ['hsn', 'sample_name', 'project_name', 'plate_name', 
                        'run_name', 'analysis_date', 'run_date', 'assembly_pass', 
-                       'percent_coverage']
+                       'percent_coverage', 'clade', 'G_clade']
     for column in columns:
```

Comment:
Test discussion.

Reply:
Test reply to discussion about [#1](https://github.com/CDPHE-bioinformatics/CDPHE-RSV/issues/1)

Reply:
More test comments.